portal_ux/README.rst

```diff
@@ -0,0 +1,69 @@
+.. |company| replace:: ADHOC SA
+
+.. |company_logo| image:: https://raw.githubusercontent.com/ingadhoc/maintainer-tools/master/resources/adhoc-logo.png
+   :alt: ADHOC SA
+   :target: https://www.adhoc.com.ar
+
+.. |icon| image:: https://raw.githubusercontent.com/ingadhoc/maintainer-tools/master/resources/adhoc-icon.png
+
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+================
+Portal Usability
+================
+
+Several Improvements:
+
+#. Add Funcionality to allow portal for companies (parents partners), by default odoo does not show parent companies on invite to portal wizard.
+#. Add a new configuration option in general setting to activate or desactivate 
+
+Installation
+============
+
+To install this module, you need to:
+
+#. Just install this module.
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+#. Go to "General Settings" section "Users" and set the boolean field "Portal in companies" to active or desactivate funcionality of portal for companies
```

Comment:
Cambiaría por:
#. To enable / disable portal users for companies: Go to "General Settings" and set/unset the field "Allow companies to be portal users"